cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -237,79 +241,56 @@ struct DispatchHistogram
     // Scales samples to evenly-spaced bins
     struct ScaleTransform
     {
+        using CommonT = typename cuda::std::common_type<LevelT, SampleT>::type;
```

Comment:
Member variables are copied on the device, could you add a static assert on `std::is_trivially_copyable<CommonT>`? Since this is going to be a part of 2.0 release, we might  break some code that relies on UB.

Reply:
Done 👍🏻 Good catch!